docs/for_trussels.md

```diff
@@ -0,0 +1,66 @@
+
+Regardless of whether you are on client work or Reserve, it is _up to each individual_ to delineate themselves as an active maintainer, and perform the following on- or off-boarding tasks as needed.
+
+### Active maintainer responsibilities include:
+
+- Maintain a presence in the #react-uswds channel, especially to answer any implementation questions from Trussels who are _not_ active maintainers
+- Attend and help facilitate the ReactUSDS check-in meetings regularly
+- Pay attention to [USWDS](https://github.com/uswds/uswds) updates, and create new issues needed to help this project stay up-to-date with them
+- Participate actively in PR reviews, issue discussion, and project roadmap planning
+- Keep an eye on any security alerts that come up, and make sure they are addressed promptly
+- Assist with shepherding library releases forward, and publishing to npm
+- Prevent PRs and issues from becoming stale, and clean up the ones that do
+- Moderate this repo and help keep it aligned with Truss project values
+
+## Onboarding
+
+First of all, we’re so excited you want to be an active maintainer on this project! Here are the things you should make sure to do:
+
+- [ ] Join the #react-uswds Slack channel
+- [ ] Add yourself to the ReactUSWDS weekly check-in (this can be found on the Truss Events calendar)
+- [ ] If you haven't already, be sure to familiarize yourself with this repo, especially the docs on [contributing](./contributing.md) and [recent releases](https://github.com/trussworks/react-uswds/releases)
+- [ ] Open a new PR that adds yourself to the Active Maintainers list in the [readme](../README.md).
+  - [ ] If you aren't already, you can also add yourself to the `contributors` list in the [package.json](../package.json),
+  - [ ] and to [CODEOWNERS](../CODEOWNERS) if you'd like to automatically be requested to review PRs _(this is optional)_.
+- [ ] _Optional:_ If you want access to the npm org so you can publish new releases, ping `@npm-admins` in Slack
+
+If you've completed all of the above and are wondering what to do next, here are some ideas!
+
+- Check out any [open pull requests](https://github.com/trussworks/react-uswds/pulls) that might need reviews or feedback
+- Take a look at the [project board](https://github.com/trussworks/react-uswds/projects/1) to get a sense of current and upcoming work
+  - Issues in the **Ready for development** column should have clear scope and acceptance criteria, and are ready to be worked on!
+    - especially any that are labeled `good first issue` if this is your first time contributing to ReactUSWDS
+  - Issues in the **Needs requirements** column are undergoing active discussion and would probably benefit from your point of view!
+- Browse [open issues](https://github.com/trussworks/react-uswds/issues) to see if there are any new issues that need to be labeled or prioritized
+
```

Comment:
I think another good onboarding step would be to check in with the other active maintainers to help get a feel for where help is needed and things along those lines :) (just an opinion though)